frontend/include/chpl/resolution/resolution-queries.h

```diff
@@ -389,6 +395,15 @@ resolveGeneratedCallInMethod(Context* context,
 */
 bool isTypeDefaultInitializable(Context* context, const types::Type* t);
 
+/**
+  Determine whether type 't' is copyable/assignable from const or/and from ref,
+  storing the results in the respective out-parameters.
```

Comment:
Can we use `bool&` for the out parameters? (FWIW, C++ implicit conversion rules allow passing an `int` say to a pointer argument; but not so to a `bool&`).